File: server/src/main/java/com/nbu/projects/dentistappointmentsys/controllers/models/EditUserProfileModel.java
package com.nbu.projects.dentistappointmentsys.controllers.models;

public class EditUserProfileModel {
    private Long id;
    private String email;
    private String firstName;
    private String lastName;

    public EditUserProfileModel(){
    }

    public EditUserProfileModel(String email, Long id, String firstName, String lastName){
        this.email = email;
        this.id=id;
       this.firstName = firstName;
        this.lastName = lastName;
    }

    public Long getId(){
        return id;
    }

   public void setId(Long id){this.id=id;}

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }
}

File: ui/src/main/ui/src/app/services/model/edit.user.profile.model.ts
export class EditPatientProfileModel{
  email: string;
  id: number;
  firstName:string;
  lastName:string;


  constructor(email: string,firstName:string,lastName:string,id:number) {
    this.email = email;
    this.firstName=firstName;
    this.lastName=lastName;
    this.id= id;
  }
}
